java/rocksjni/options.cc

```diff
@@ -1180,7 +1179,7 @@ jbyte Java_org_rocksdb_Options_compactionStyle(
 void Java_org_rocksdb_Options_setMaxTableFilesSizeFIFO(
     JNIEnv* env, jobject jobj, jlong jhandle, jlong jmax_table_files_size) {
   reinterpret_cast<rocksdb::Options*>(jhandle)->compaction_options_fifo.max_table_files_size =
-    static_cast<long>(jmax_table_files_size);
+    static_cast<uint64_t>(jmax_table_files_size);
```

Comment:
I know, but casting int64 (jlong) to long is not correct. One will loose upper 32 bits which was correctly discovered by unit test. This change is fix for that.